experiments/modelfree/plot_noisy_observations.py

```diff
@@ -0,0 +1,141 @@
+import json
+import math
+import os
+
+from matplotlib import pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+RESULT_DIR = "/Users/cody/Data/adversarial_policies/ray_results/noisy_victim_obs/"
```

Comment:
Please don't use absolute directories :( We should aim to make the code run out of the box on anyone's machine.

The convention I've followed is to assume the CWD is the root of the project, and that the `data` subdirectory contains all results. If you upload your results to S3, then you can assume they exist in `data/aws/` -- which they will if someone has run the AWS sync script, and is easy for them to do if not.

Also I'm confused why the data lives in `ray_results`? That is just a temporary directory, and it should always be copied out of there at the end of the experiment.

Reply:
ray_results is just a directory elsewhere on my machine, not the temporary one you're thinking of. 

It generally felt more fault-tolerant to me to store results elsewhere on my machine rather than trying to reason about the whole complex set of code expectations about what was and wasn't expected to be in /data/aws at any given time, but I can refactor it.

Reply:
I refactored to both the experiment directory and an out-directory to be within /data/aws, but it seems probable that the way I've done it will go against what you want in those directories somehow, in which case I can change it.